_themes/sphinx_rtd_theme/footer.html

```diff
@@ -45,7 +45,7 @@
   {%- if show_sphinx %}
   {% trans %}Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>{% endtrans %}.
   {%- endif %}
-
+  <p>Deployed with <a href="https://platform.sh" title="Platform.sh">Platform.sh</a></p>
```

Comment:
Yes, I'm waiting for assets :) for now platform.sh GitHub integration don't works :(